tests/test_cli.py

```diff
@@ -1,26 +1,37 @@
-# -*- coding: utf-8 -*-
-import os
-
-from pydeps.pydeps import pydeps
-from tests.filemaker import create_files
-from tests.simpledeps import simpledeps, empty
-
-
-def test_output(tmpdir):
-    files = """
-        unrelated: []
-        foo:
-            - __init__.py
-            - a.py: |
-                from bar import b
-        bar:
-            - __init__.py
-            - b.py
-    """
-    with create_files(files) as workdir:
-        assert os.getcwd() == workdir
-
-        outname = os.path.join('unrelated', 'foo.svg')
-        assert not os.path.exists(outname)
-        pydeps(fname='foo', **empty('--noshow', output=outname))
-        assert os.path.exists(outname)
+# -*- coding: utf-8 -*-
+import os
+from unittest.mock import patch, call
+
+from pydeps.cli import error
+from pydeps.pydeps import pydeps
+from tests.filemaker import create_files
+from tests.simpledeps import simpledeps, empty
+
+
+def test_output(tmpdir):
+    files = """
+        unrelated: []
+        foo:
+            - __init__.py
+            - a.py: |
+                from bar import b
+        bar:
+            - __init__.py
+            - b.py
+    """
+    with create_files(files) as workdir:
+        assert os.getcwd() == workdir
+
+        outname = os.path.join('unrelated', 'foo.svg')
+        assert not os.path.exists(outname)
+        pydeps(fname='foo', **empty('--noshow', output=outname))
+        assert os.path.exists(outname)
+
+@patch('sys.exit')
```

Comment:
maybe pytest's `monkeypatch` (https://docs.pytest.org/en/6.2.x/monkeypatch.html) and `capsys` (https://docs.pytest.org/en/6.2.x/capture.html) would be easier to work with across Python versions..?

Reply:
Thanks, refactored to use capsys for checking the print calls and a try/catch block for the sys.exit call.  Let me know if you think it would be clearer / more consistent to use monkeypatch for sys.exit?

Reply:
Also, is a version bump required here?  I'm not sure whether pydeps follows semver or not.